pkg/platform/platform.go

```diff
@@ -22,6 +22,15 @@ import (
 	"github.com/kinvolk/lokomotive/pkg/version"
 )
 
+// CommonControlPlaneCharts defines a list of control plane Helm charts to be deployed for all
+// platforms.
+var CommonControlPlaneCharts = []string{
```

Comment:
We should not use global variables. If slice is needed, I'd use function instead, as then it makes the values immutable.